sample/RazorApp/Pages/About/Index.temp.cshtml

```diff
@@ -0,0 +1,8 @@
+﻿@page
```

Comment:
Need to study how razor page route to pages. Or responsive can't work the razor pages.